build_cffi.py

```diff
@@ -51,6 +40,23 @@
         "of {}".format(available_levels)
     )
 
+# =================================================================
+# Set compilation arguments dependent on environment... a bit buggy
+# =================================================================
+extra_compile_args = ["-fopenmp", "-Wall", "--verbose", f"-DLOG_LEVEL={log_level:d}"]
+if "DEBUG" in os.environ:
+    extra_compile_args += ["-g", "-O0"]
+else:
+    extra_compile_args += ["-Ofast"]
+
+libraries = ["m", "gsl", "gslcblas", "fftw3f_omp", "fftw3f"]
```

Comment:
Did you pull through the clang changes on master?

Reply:
Not yet, I'll merge them in soon